python/cuml/utils/numba_utils.py

```diff
@@ -13,9 +13,7 @@
 # limitations under the License.
 #
 
-import numba
-import math
-
+import cupy as cp
```

Comment:
Probably cudf_utils or something would be a better place for this as it is no longer using numba?